editor/script_editor_debugger.cpp

```diff
@@ -574,7 +574,38 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, const Array &p_da
 		for (int i = 0; i < arr.size(); i++) {
 			p[i] = arr[i];
 			if (i < perf_items.size()) {
-				perf_items[i]->set_text(1, rtos(p[i]));
+
+				float v = p[i];
+				String vs = rtos(v);
+				String tt = vs;
+				switch (Performance::MonitorType((int)perf_items[i]->get_metadata(1))) {
+					case Performance::MONITOR_TYPE_MEMORY: {
+						// for the time being, going above GBs is a bad sign.
+						String unit = "B";
```

Comment:
this should be appended afterward?

Reply:
yeah, i think that would make it better.